cohttp-lwt-jsoo/src/cohttp_lwt_jsoo.ml

```diff
@@ -107,12 +107,12 @@ module Body_builder (P : Params) = struct
           (fun () -> `String (Js.string ""))
           (fun s -> `String s)
       in
-      if xhr_response_supported then
-        if Js.Opt.return xml##.response == Js.null then (
```

Comment:
The only worth change is `==` to `=` here. The rest is just because I find a match easier to read than nested if then elses and I am happy to drop it if deemed unnecessary.

Reply:
@mefyl was there a specific reason you used the physical equality `==` instead of `=`?

Reply:
Yes, I wanted `response === null` in javascript. It checks the value is actually the null singleton, the other alternative would also catch `undefined` (and would thus be marginally slower). It's a common stylistic precept in js to use `===` in most places, even strings which are flyweight instances. So the alternative will probably work the same,  but I think it's marginally better stylistically (juste like you should probabl `==` between two `Js.js_string Js.t`).

Reply:
Ok thanks for the explanation (which I stole for the commit message 😄 ) Then I revert it!